README.md

```diff
@@ -0,0 +1,172 @@
+
+The following environment variables are available for all processes:
+
+| Name | Required? | Default | Description |
+| ---- | --------- | ------- | ----------- |
+| `AWS_DEFAULT_REGION` | :white_check_mark: | | Region for connecting to AWS |
+| `DATABASE_URL` | :white_check_mark: | | URL with connection data for the database. Typically a postgres URL of the form `postgres://user:password@host/database_name`. |
+| `DJANGO_SECRET_KEY` | :white_check_mark: | | Secret key for Django's cryptographic signing. We don't really use it but Django fails to start without it. Set to a secret, random string. |
+| `SENTRY_DSN` | :white_check_mark: | | DSN used to send errors to Sentry. |
+| `AWS_CONNECT_TIMEOUT` | :x: | `30` | Timeout for connecting to AWS |
+| `AWS_READ_TIMEOUT` | :x: | `30` | Timeout for reading a response from AWS |
+| `LOG_FORMAT` | :x: | `mozlog` | Format to output logs in. Use `mozlog` for mozlog-compatible logs, or `compose` for a human-readable format suited for docker-compose output. |
+| `LOG_LEVEL` | :x: | `INFO` | Minimum log level to output. One of `CRITICAL`, `ERROR`, `WARNING`, `INFO`, or `DEBUG`. |
+
+### Processor
+
+Command: `bec-alerts processor`
+
+Reads from SQS to fetch incoming events from Sentry, processes them, and saves the resulting data to Postgres. This process will launch and manage a pool of subprocesses for listening and processing data.
+
+The following environment variables are available for the processor:
+
+| Name | Required? | Default | Description |
+| ---- | --------- | ------- | ----------- |
+| `PROCESSOR_SLEEP_DELAY` | :x: | `20` | Seconds to wait between polling the queue |
+| `SQS_QUEUE_NAME` | :x: | `sentry_errors` | Name of the queue to poll for events. |
+| `SQS_ENDPOINT_URL` | :x: | | Endpoint URL for connection to AWS. Only required for local development. |
+| `PROCESSOR_PROCESS_COUNT` | :x: | System CPU count | Number of worker processes to start |
+| `PROCESSOR_WORKER_MESSAGE_COUNT` | :x: | `200` | Number of messages a worker should process before terminating; a new worker process is started to take its place. Workers may process slightly more messages than this number before shutting down due to message batching. |
+
+
+### Watcher
+
+Command: `bec-alerts watcher`
+
+Periodically checks for events that have been processed since the last run, and evaluates alert triggers to determine if we should send a notification to users. This process implements its own sleep timer.
+
+The following environment variables are available for the watcher:
+
+| Name | Required? | Default | Description |
+| ---- | --------- | ------- | ----------- |
+| `SES_FROM_EMAIL` | :white_check_mark: | `notifications@sentry.prod.mozaws.net` | Email to use in the From field for notifications |
+| `DATADOG_API_KEY` | :white_check_mark: | | API key for incrementing the healthcheck counter in Datadog via DogStatsD. |
```

Comment:
Are you sure you need this? We have a Datadog Agent configured on the host, so I think all you need to do is talk statsd to it on `localhost:8125`

Reply:
It's probably unnecessary.